src/data/moves/move-attrs/ability-copy-attr.ts

```diff
@@ -62,4 +63,26 @@ export class AbilityCopyAttr extends MoveEffectAttr {
       return ret;
     };
   }
+
+  /**
+   * If the target's ability is detrimental, grants a (-5) penalty.
+   * Otherwise, grants (+2) effect score for each Pokemon whose detrimental ability is overridden
+   * by this effect. The total bonus cannot exceed (+3).
```

Comment:
```suggestion
   * Otherwise, grants (+2) effect score if one Pokemon's detrimental ability is overridden
   * by this effect, and (+3) if more than one Pokemon's detrimental ability is overridden.
```
Might as well be more explicit on how it works.